python/cuml/test/test_nearest_neighbors.py

```diff
@@ -232,14 +236,14 @@ def test_knn_separate_index_search(input_type, nrows, n_feats, k, metric):
         D_cuml_arr = D_cuml.as_gpu_matrix().copy_to_host()
         I_cuml_arr = I_cuml.as_gpu_matrix().copy_to_host()
     else:
-        assert isinstance(D_cuml, np.ndarray)
-        assert isinstance(I_cuml, np.ndarray)
-        D_cuml_arr = D_cuml
-        I_cuml_arr = I_cuml
+        assert isinstance(D_cuml, cp.core.core.ndarray)
```

Comment:
It's nice that now both paths have GPU data (we already had cudf above, now we have cupy below) so while the .get()s are a pain in some ways, this is more consistent.